lecture_1/homework/app/controllers/samurais_controller.rb

```diff
@@ -0,0 +1,53 @@
+class SamuraisController < ApplicationController
+  def show
+      render json: samurai.to_json(
+        only: %w[id name cuirass_quality battles_count joined_date])
+  end
+
+  def index
+    if params[:dead].present?
+      params[:dead].to_i == 1 ?
```

Comment:
Operatory tenarne nie wyglądają już ładnie gdy rozbijają się na wiele linijek :D Lepiej chyba w takich sytuacjach przerabiać to na zwykłego ifa :)